templates/4b_normalize_image_data.ipynb

```diff
@@ -27,9 +27,9 @@
     "sys.path.append('../')\n",
```

Comment:
<p>What happens if the function runs on an already formatted panel, like this one? Will be good to check and make sure it doesn't cause problems</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I added a check to `convert_panel()` that will simply append '-toffy' to the file name (for simpler detection in any other notebooks) and then returns the already formatted panel data

Reply:
Right, but maybe the user renames it, or they run it on the example panel, etc. 

We should just make sure that if the name matching fails, the function still runs as expected on already formatted panel